packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -625,102 +542,90 @@ function insertNodesToEditableTree(
-	assert(node !== undefined);
 	switch (shape) {
 		case TreeShape.Deep:
-			treeRoot[createField](localFieldKey, singleTextCursor(node));
+			treeRoot[createField](localFieldKey, singleTextCursor(testTreeNode));
 			assert(isUnwrappedNode(treeRoot));
 			field_0 = treeRoot[getField](localFieldKey);
 			assert(field_0 !== undefined);
 			currentNode = field_0.getNode(0);
 			for (let i = 0; i < numberOfNodes; i++) {
 				assert(isUnwrappedNode(currentNode));
-				currentNode[createField](localFieldKey, singleTextCursor(node));
+				currentNode[createField](localFieldKey, singleTextCursor(testTreeNode));
 				currentNode = currentNode[getField](localFieldKey).getNode(0);
 			}
 			break;
 		case TreeShape.Wide:
 			assert(isUnwrappedNode(treeRoot));
-			for (let i = 0; i < numberOfNodes - 1; i++) {
-				treeRoot[getField](localFieldKey).insertNodes(i, singleTextCursor(node));
+			for (let i = 0; i < numberOfNodes; i++) {
+				treeRoot[getField](localFieldKey).insertNodes(i, singleTextCursor(testTreeNode));
 			}
 			break;
 		default:
 			unreachableCase(shape);
 	}
 }
 
-function getTestTreeAsJSObject(
-	numberOfNodes: number,
-	shape: TreeShape,
-	dataType: string,
-): Jsonable {
+function getTestTreeAsJSObject(numberOfNodes: number, shape: TreeShape): Jsonable {
 	let tree;
 	switch (shape) {
 		case TreeShape.Deep:
-			tree = [getJSTestTreeDeep(numberOfNodes, dataType)];
+			tree = [getJSTestTreeDeep(numberOfNodes)];
 			break;
 		case TreeShape.Wide:
-			tree = getJSTestTreeWide(numberOfNodes, dataType);
+			tree = getJSTestTreeWide(numberOfNodes);
 			break;
 		default:
 			unreachableCase(shape);
 	}
 	return tree;
 }
 
-function getJSTestTreeWide(numberOfNodes: number, dataType: string): Jsonable {
+function getJSTestTreeWide(numberOfNodes: number): Jsonable {
 	const nodes = [];
-	const node = testSubtrees.get(dataType);
 	for (let i = 0; i < numberOfNodes - 1; i++) {
-		nodes.push(node);
+		nodes.push(testTreeNode);
 	}
 	const tree = {
 		type: dataSchema.name,
 		fields: {
 			foo: nodes,
 		},
-		value: testSubtrees.get(dataType)?.value,
+		value: testTreeNode.value,
 	};
 	return tree;
 }
 
-function getJSTestTreeDeep(numberOfNodes: number, dataType: string): Jsonable {
-	const node = testSubtrees.get(dataType);
-	assert(node !== undefined);
+function getJSTestTreeDeep(numberOfNodes: number): Jsonable {
 	if (numberOfNodes === 1) {
-		return {
-			type: dataSchema.name,
-			value: node.value,
-		};
+		return testTreeNode;
 	}
 	const tree = {
 		type: dataSchema.name,
 		fields: {
-			foo: [getJSTestTreeDeep(numberOfNodes - 1, dataType)],
+			foo: [getJSTestTreeDeep(numberOfNodes - 1)],
 		},
-		value: node.value,
+		value: testTreeNode.value,
 	};
 	return tree;
 }
 
-function readTreeAsJSObject(tree: Jsonable, initialTotal: Jsonable, dataType: string): Jsonable {
-	let currentTotal = initialTotal as number | string | boolean | JsonableTree;
+function readTreeAsJSObject(tree: Jsonable, initialTotal: Jsonable): Jsonable {
```

Comment:
I think it would be better to remove the initialTotal argument. Instead below do `currentTotal += readTreeAsJSObject`

THis makes what is going on with it more clear, and avoids needing an extra paramater.

If you do want to keep this parameter, it should be typed number not Jsonable (also change the type for the return value)